system/jhtree/ctfile.hpp

```diff
@@ -92,7 +98,7 @@ struct __declspec(novtable) jhtree_decl KeyHdr
     __int64 fileSize; /* fileSize - was once used in the bias calculation e0x */
     short nodeKeyLength; /* key length in intermediate level nodes e8x */
     unsigned short version; /* build version - to be updated if key format changes    eax*/
-    short unused[2]; /* unused ecx */
+    unsigned keyFlags; /* extended version of ktype ecx */
```

Comment:
This structure probably should be defined with pack(1). Without it, I had to think to convince myself that this was ok (also we should really be using types with determinate sizes, rather than unsigned whose size is implementation-defined).